ui/src/context/RolesContext.tsx

```diff
@@ -118,9 +120,9 @@ function useRoleKinds(): Set<RoleKind> {
 function useRolesContext() {
   const context = React.useContext<RolesState>(RolesStateContext);
   if (context === undefined) {
-    throw new Error('useRoles must be used within a RolessProvider');
+    throw new Error('useRoles must be used within a RolesProvider');
   }
   return context;
 }
 
-export { RolesProvider, useRolesContext, useRoleKinds };
```

Comment:
I feel like if we do keep it we should leave it exported? So that this file doesnt have to change if it is used in the future?